backend/yarn.lock

```diff
@@ -447,6 +474,12 @@ fresh@0.5.2:
   resolved "https://registry.yarnpkg.com/fresh/-/fresh-0.5.2.tgz#3d8cadd90d976569fa835ab1f8e4b23a105605a7"
   integrity sha1-PYyt2Q2XZWn6g1qx+OSyOhBWBac=
 
+<<<<<<< HEAD
```

Comment:
make sure to delete these merge conflict tags

Reply:
oops, thats me

Reply:
wait brandon they're still there